providers/agent/mcorpc/audit/choria.go

```diff
@@ -41,12 +44,20 @@ func Request(request protocol.Request, agent string, action string, data json.Ra
 	}
 
 	logfile := cfg.Option("plugin.rpcaudit.logfile", "")
+	logfileGroup := cfg.Option("plugin.rpcaudit.logfile.group", "")
+	logfileModeOpt := cfg.Option("plugin.rpcaudit.logfile.mode", "0600")
```

Comment:
Probably worth adding all these 3 to teh config classes as options so that they are discoverable using things like `choria tool config` and the generated docs. 

I can deal with that in a follow up as looks like there might be some annoyance with getting that in.  So just a comment here for now that I'll do that later.